src/create_order.rs

```diff
@@ -86,64 +74,74 @@ impl Contract {
         };
 
         ext_token::ext(sell_token.clone())
-            .with_static_gas(Gas(3))
+            .with_static_gas(Gas(200))
```

Comment:
is it too much gas?

Reply:
there was some issued with exceeding prepaid gas so i set up that high value of Gas, we can make it lower later, but for now let it be